app/src/main/java/com/example/beerapp/BeerListConnector.java

```diff
@@ -1,18 +1,26 @@
 package com.example.beerapp;
 
+import com.example.beerapp.API.ApiService;
+import com.example.beerapp.API.RetrofitConnector;
+import com.example.beerapp.Model.BeerList;
+
+import io.reactivex.Observable;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
 import retrofit2.Retrofit;
-import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
-import retrofit2.converter.gson.GsonConverterFactory;
 
 public class BeerListConnector implements BeerListRepository {
     @Override
-    public Retrofit getNetworkData() {
-        Retrofit retrofit = new Retrofit.Builder()
-                .baseUrl("https://sandbox-api.brewerydb.com/v2/")
-                .addConverterFactory(GsonConverterFactory.create())
-                .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
-                .build();
+    public void getNetworkData() {
+        Retrofit retrofit = RetrofitConnector.makeConnect();
```

Comment:
The repository don\`t have to know about instance of retrofit. You can only have `ApiService`